src/docs/getting-started.md

```diff
@@ -67,11 +67,46 @@ To write an essay, copy one of the templates and edit that copy.
 3. In the pop-up menu that appears, select **Duplicate**.
 
 This creates a new notebook, with the name of the template followed by '-2', and opens it.
-You can now start editing your copy of the template.
-(Once you decided your essay's topic, you can
-[rename the notebook](deepnote-how-to.md#rename-duplicate-download-or-delete-a-notebook-or-file).)
+
+Once you decided your essay's topic, you can
+[rename the notebook](deepnote-how-to.md#rename-duplicate-download-or-delete-a-notebook-or-file).
+
+## Write the essay
+
+Now that you have copied and renamed a template you can start writing your essay in Deepnote.
 
 If you're familiar with the classic Jupyter interface, we recommend you first
 read about the [differences](deepnote-background.md#deepnote-vs-classic-notebook) with Deepnote.
 
 For a video introduction to notebooks and Deepnote, see our [Deepnote guide](deepnote.md).
+
+For guidance on writing algorithmic essays see our [writing](writing.md) and
+[coding](coding.ipynb) guides.
+
+## Share the essay
+
+When you are ready to share the essay with others, do the following.
+
+1. [Enabled sharing](deepnote-how-to.md#enable-sharing) for your workspace.
+2. Click the **Share** button in the top right corner of the screen.
+3. To the right of **Anyone with a link to this project**, click on the drop-down menu and select **Comment**.
+4. Click the highlighted blue **link to this project**, to copy the link.
+5. Share that link with your peers by email or by posting in your course's forum.
+
+## Comment on essays
+
+After sharing your essay, others will be able to comment on it. You can also make comments on essays as part of the feedback process.
```

Comment:
I think this first line is confusing as it muddles those who share and those are going to comment. Since the section is about commenting, I suggest starting with something like

Once an essay has been shared with you, you can comment on it. See our guide on how to [leave feedback](link to giving comments section). If you have commenting permission, ...